.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: Test PyEasyArchive
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
```

Comment:
Can we use debian bookworm as a target?

Reply:
I didn't see debian as an option in the hosted runners: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners